ember-flight-icons/tests/acceptance/index-test.js

```diff
@@ -14,13 +14,13 @@ module('Acceptance | icon index and percy-test', function (hooks) {
     await waitFor('.ds-icon-frame > .flight-icon', { timeout: 1000 });
     assert.dom('[data-test-target="icon-grid"] [data-test-icon]').exists();
 
-    await percySnapshot('Icons page');
+    await percySnapshot('Icons page', { enableJavaScript: true });
```

Comment:
@amyrlam FYI when I run the same local command against the `main` branch, the icons are visible and correctly rendered: https://percy.io/HashiCorp/flight/builds/13131356/changed/741687065?browser=chrome&browser_ids=20&subcategories=approved&viewLayout=side-by-side&viewMode=new&width=1280&widths=375%2C1280

Reply:
👍 forsure good call. Mentioned that a bit ago in the support email thread. But, since there's been some confusion in that thread (they never really responded to that aspect) ... shared the minimal repro

Hopefully they'll have something soon!